test/projects/file-generate/deploy/crds/mygroup.com_testkinds_crd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
It would be cool if `op-generate` had a real effect as well as `go-generate`. E.g. remove this file, and prove that it gets generated by your test.

Reply:
✓ #59